content/docs/catalogue/items.md

```diff
@@ -98,6 +98,32 @@ There are two tabs in the bottom part of the screen:
 This tab lists the Master Lists associated with this item.
 ![Item detail headers](/docs/catalogue/images/cat_itemmasterlists.png)
 
+### Ledger
+
+The item ledger tab shows the history of the item.
```

Comment:
:nit a history of the item could be 'I changed default pack size to..'
whereas the ledger is a record of stock transfers for the store
```suggestion
The item ledger tab shows a history of transactions in this store for the item.
```